dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -60,6 +59,7 @@
     <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
   <Import Condition="'$(Configuration)' == 'Release'" Project="$(MSBuildProjectDirectory)\..\..\PGO.compile.props" />
+  <Import Condition="'$(PGOBuildMode)' == 'Optimize'" Project="$(MSBuildProjectDirectory)\..\..\tools\MUXPGODatabase\PGO.version.props" />
```

Comment:
We should make it so that we always optimize for 'Release' configuration.

Reply:
+1. We don't have to instrument always but we can optimize always. It might take a little bit longer to build, but the tests should run a little faster and it helps us find any PGO issues earlier.

Reply:
There is a fallback in PGO.compile.props which sets optimize if PGOBuildMode is not set.  In that case we need to not pass it to build command line as a parameter.

Reply:
I changed it so PGOBuildMode is set on right configuration in yml files and global pipeline variables are not used.  I will remove that variable from PR pipeline as well.